mobile/src/components/uam-configure-group/components/users-table/users-table.tsx

```diff
@@ -21,23 +20,22 @@ type Props = {
     page: number;
     setPage: (page: number) => void;
   };
+  checkedIds: number[];
 };
 
-const UsersTable: FC<Props> = ({ users, onCheckboxToggle, pagination }) => {
-  const { control, reset } = useAppForm({ defaultValues: {} });
+const UsersTable: FC<Props> = ({
+  users,
+  onCheckboxToggle,
+  pagination,
+  checkedIds,
+}) => {
   const userRows = getUserRows({
     users: users.items,
     onToggle: onCheckboxToggle,
-    control,
+    checkedUserIds: checkedIds,
   });
   const userColumns = getUserColumns();
 
-  useFocusEffect(
-    useCallback(() => {
-      return () => reset({});
```

Comment:
Can't we just set values for checkboxes here? Without additional logic

Reply:
I've tried, but faced with a bug, when I open for editing one group, cancel, and open another one, the previous items are still selected, until I refresh the page, or use pagination.

Reply:
I think the problem that there we return callback that clears form when the screen loses focus, but if we need to fill data when we open screen then we need to do this inside callback, not return this function